src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs

```diff
@@ -1969,27 +1975,32 @@ private Task WriteRowSourceToServerAsync(int columnCount, CancellationToken ctok
                 if (resultTask != null)
                 {
                     finishedSynchronously = false;
-                    return resultTask.ContinueWith((t) =>
-                    {
-                        try
+                    return resultTask.ContinueWith(
+                        static (Task t, object state) =>
```

Comment:
NIT: in some places you have chosen `(Task t, object state)` and in some other places `(t, state)`. Can we use one  convention?

Reply:
I tried to standardize on `(Task t, object state)`, can you point out where I missed?

Reply:
Those were not necessarily from your changes, but were included. We can look at them later we have more time I think.